projects/ci-cd.md

```diff
@@ -0,0 +1,85 @@
+# CI/CD Observability Semantic Conventions Working Group
+
+## Description
+
+The goal of this working group is define the semantic conventions for CI/CD Observability and outline 
+a path forward for the OpenTelemetry community. 
+
+### Why
+
+The goal of this project is to establish standard semantic conventions for Continuous
+Integration (CI) and Continuous (CD) observability. This will provide a common language
+and standardized formats for CI/CD observability, enabling the community to observe CI/CD systems.
+
+This will broaden the target audience of OpenTelemetry to Release Engineering and DevOps teams,
+further cementing OpenTelemetry as the industry standard Observability framework.
+
+The timing is ripe to start now. The [CI/CD Observability OTEP](https://github.com/open-telemetry/oteps/pull/223/files)
+has been open since January of 2023 and with the recent changes to the OTEP process, the KubeCon talk, and vendor acknowledgements, there's momentum available to carry this forward. The industry 
+is heavily looking for solutions and watching the related OTEP with interest.
+
+## Deliverables
+
+The CI/CD observability working group is planning to deliver the [CI/CD Observability OTEP](https://github.com/open-telemetry/oteps/pull/223/files) with prototypes in at least two major vendors:
+
+* [GitHub](https://github.com)
+* [GitLab](https://gitlab.com)
+* [Jenkins](https://www.jenkins.io/)
+* [Keptn](https://lifecycle.keptn.sh/)
+
+A related OTEP that is of great interest to the CI/CD OTEP is that of [Environment Variables as Context Propagators](https://github.com/open-telemetry/opentelemetry-specification/issues/740) which this working group would be available to help with.
```

Comment:
Note: We're defining this as related currently, but may want to remove as it's not tightly coupled. Commenting here to open up for discussion on thoughts.